characters.txt

```diff
@@ -1,4 +1,12 @@
-                                                                                                   1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000222
-         1111111111222222222233333333334444444444555555555566666666667777777777888888888899999999990000000000111111111122222222223333333333444444444455555555556666666666777777777788888888889999999999000
+# Remember to try different line wrap modes
+# Becareful about your viewer wrapping around white spaces
+
+# 1s place
 1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012
-         10        20        30        40        50        60        70        80        90        100       110       120       130       140       150       160       170       180       190       200
+
+# 10s place         
+0000000001111111111222222222233333333334444444444555555555566666666667777777777888888888899999999990000000000111111111122222222223333333333444444444455555555556666666666777777777788888888889999999999000
+0        10        20        30        40        50        60        70        80        90        100       110       120       130       140       150       160       170       180       190       200
+
+# 100s place
+0                                                                                                  100                                                                                                 200
```

Comment:
There is a "visual bug" with GitHub. GitHub doesn't align the first character in lines which are edited and not edited.

![image](https://user-images.githubusercontent.com/1182/122663000-dc532400-d1c9-11eb-9eaf-f85bbb304e50.png)

Notice that the first character on:

- Line 4 is `#`
- Line 5 is `1`

and they aren't aligned